src/utils/word/index.js

```diff
@@ -0,0 +1,23 @@
+export function selectWords(num, length) {
+    const data = require('../../data/words/' + length + '_letters_normal.json');
+    const ret = [];
+    const obj_keys = Object.keys(data);
+    const used_keys = [];
+
+    while(ret.length < num) {
+        var ran_key = obj_keys[Math.floor(Math.random() *obj_keys.length)];
+        if (!used_keys.includes(ran_key)){
+            used_keys.push(ran_key);
```

Comment:
So you don't actually persist `used_keys` anywhere , so after this runs once it's gonna get tossed out?  Or is the purpose to find "n" unique items into your list of selected words? Couldn't you just toss out `used_keys` and check `!ret.includes(ran_key)` ?

Also, should `ran_key` be `potential_key`? 

Also also, I appreciate your usage of snake case :)

Reply:
Oh yeah `used_keys` is totally redundant here good catch. `ran_key` is supposed to be short for `random_key` but I suppose `potential_key` would be more descriptive.